content/intro-to-storybook/react/zh-CN/creating-addons.md

```diff
@@ -0,0 +1,482 @@
+---
+title: '奖励：创建一个插件'
+tocTitle: '奖励：创建一个插件'
+description: '了解如何构建自己的插件，这些插件将为您的开发增添更多动力'
+commit: 'bebba5d'
+---
+
+之前，我们介绍了 Storybook 的一个关键功能，就是强大的 [插件](https://storybook.js.org/docs/react/configure/storybook-addons) 生态系统。插件用于增强你的开发体验和工作流程。
+
+在这一章中，我们将介绍如何创建自己的插件。您可能会认为编写它可能是一项艰巨的任务，但实际上并非如此，我们只需要采取几个步骤就可以开始编写它。
+
+但是首先，让我们先确定插件的作用。
+
```

Comment:
@STPace recently #396 was merged and it included  some small changes to this section. If you don't mind, could you adjust the translation accordingly?  So that you get the most current content?

Reply:
Updated 🎉